pkgs/tools/misc/present-cli/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "present";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "terror";
+    repo = pname;
+    rev = version;
+    sha256 = "+kCHe84ikdCLd7j5YwP2j3xz+XTzzo/kLy+b9YUFDnI=";
+  };
+
+  cargoSha256 = "VKY/FQUrFWtLxKoK6LP6qPMqNN4absZvnAbH9mha1fI=";
+
+  # required for tests
+  preBuild = ''
+    patchShebangs bin/get_version
```

Comment:
This is normally done in the `postPatch` phase.

Reply:
Thanks, updated.